projects/cdk/observables/zone-free.ts

```diff
@@ -28,7 +13,10 @@ export function tuiZonefull<T>(ngZone: NgZone): MonoTypeOperatorFunction<T> {
 }
 
 export function tuiZonefree<T>(ngZone: NgZone): MonoTypeOperatorFunction<T> {
-    return source => source.lift(new TuiZonefreeOperator<T>(ngZone));
```

Comment:
Hi, @waterplea!

Could you please explain why you changed this?

Reply:
lift is deprecated in rxjs 7 and will be made private in rxjs 8. Frankly, not sure why we didn't do it this way from the get go, but that wasn't my code when it was first written and back then I wasn't that proficient with rxjs 🙂